spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -651,6 +682,31 @@ def setup_screenshots_for(row)
         return result
       end
 
+      # generates the nested data structure to represent screenshots
+      def setup_messages_screenshots_for(row)
+        return [] if row.nil? || row["displayFamilies"].nil?
+
+        display_families = row.fetch("displayFamilies", {}).fetch("value", nil)
+        return [] unless display_families
+
+        result = []
+
+        display_families.each do |display_family|
+          display_family_screenshots = display_family.fetch("messagesScreenshots", {})
+          next unless display_family_screenshots
```

Comment:
Unless the value of display_family screenshots is explicitly set to `nil` in the hash, this line never triggers because of the `fetch` above